### PR TITLE
Add facebook meta tags

### LINKED ISSUE #40: add facebook meta tags
lifted from mozilla.org:

``` html
    <meta property="og:type" content="website">
    <meta property="og:site_name" content="Mozilla">
    <meta property="og:locale" content="en_US">
    <meta property="og:url" content="https://www.mozilla.org/en-US/">
    <meta property="og:image" content="https://mozorg.cdn.mozilla.net/media/img/mozorg/mozilla-256.jpg">
    <meta property="og:title" content="Home of the Mozilla Project">
    <meta property="og:description" content="">
    <meta property="fb:page_id" content="262134952380">
```

### PR DESCRIPTION
**Pull request by FernandaBernardo**
closes #40

### CODE REVIEW

File: public_html/index.html [hunk 1]
```diff
@@ -15,6 +15,14 @@
     <!--[if lt IE 9]>
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla - The Future of Web Games">
+    <meta property="og:locale" content="en_US">
+    <meta property="og:url" content="https://games.mozilla.org/">
+    <meta property="og:image" content="https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg">
```
Comment (cvan) [inline public_html/index.html]: do you need the secure og:image too? last time I tested, for some reason, I did

File: public_html/index.html [hunk 2]
```diff
@@ -15,6 +15,14 @@
...
+    <meta property="og:title" content="Mozilla - The Future of Web Games">
+    <meta property="og:description" content="">
```
Comment (cvan) [inline public_html/index.html]: I would probably use the description from above

File: public_html/index.html [hunk 3]
```diff
@@ -15,6 +15,14 @@
     <!--[if lt IE 9]>
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla - The Future of Web Games">
+    <meta property="og:locale" content="en_US">
+    <meta property="og:url" content="https://games.mozilla.org/">
+    <meta property="og:image" content="https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg">
+    <meta property="og:title" content="Mozilla - The Future of Web Games">
```
Comment (cvan) [inline public_html/index.html]: perhaps make this `games.mozilla.org`?

File: public_html/index.html [hunk 4]
```diff
@@ -15,6 +15,14 @@
     <!--[if lt IE 9]>
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla - The Future of Web Games">
+    <meta property="og:locale" content="en_US">
```
Comment (cvan) [inline public_html/index.html]: I like that you added the locale. good catch l!

File: public_html/index.html [hunk 5]
```diff
@@ -15,6 +15,14 @@
     <!--[if lt IE 9]>
       <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
+    <meta property="og:type" content="website">
+    <meta property="og:site_name" content="Mozilla - The Future of Web Games">
```
Comment (cvan) [inline public_html/index.html]: a recent thing I learned:

if you define these OpenGraph tags, you actually don’t need to define a lot (most?) for the Twitter Card tags.

that might make your other PR a bit simpler. YMMV.

Comment (FernandaBernardo) [reply public_html/index.html]: I didn't know about this. It's very nice! But what meta tags I need to remove from the twitter PR?

File: public_html/index.html (hunk 4 above)
Comment (FernandaBernardo) [reply public_html/index.html]: *-*

File: public_html/index.html (hunk 1 above)
Comment (FernandaBernardo) [reply public_html/index.html]: ohh, i really don't know what the best in this case....

File: public_html/index.html (hunk 3 above)
Comment (FernandaBernardo) [reply public_html/index.html]: but this is the title of the card
image

I think it is weird to be the url. What do you think?

Comment (FernandaBernardo) [reply public_html/index.html]: done!

File: public_html/index.html (hunk 2 above)
Comment (FernandaBernardo) [reply public_html/index.html]: done!

File: public_html/index.html (hunk 1 above)
Comment (cvan) [reply public_html/index.html]: I'm actually not entirely sure. IME, docs don't always match implementation.

File: public_html/index.html [hunk 6]
```diff
@@ -15,6 +15,14 @@
...
+    <meta property="og:locale" content="en_US">
+    <meta property="og:url" content="https://games.mozilla.org/">
+    <meta property="og:image" content="https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg">
+    <meta property="og:description" content="Mozilla - The Future of Web Games">
+    <meta property="og:title" content="games.mozilla.org">
```
Comment (cvan) [inline public_html/index.html]: you can actually remove this tag. `og:title` I guess is mainly used for individual articles/posts. (news to me!)

Comment (cvan) [reply public_html/index.html]: I'm just going to manually edit this so we can merge this - save you some hassle 😄

**Comment by cvan** [comment]
thanks for this!

**Pull request merged**